releasenotes/notes/2.1/fix-assign_parameters-string-input-with-strict-false-5597010d6d0c4a5f.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixed a bug in method :meth:`.QuantumCircuit.assign_parameters` that results in error with string inputs
```

Comment:
```suggestion
    Fixed a bug in method :meth:`.QuantumCircuit.assign_parameters` that results in an error when there are string inputs
```

Reply:
I couldn't tell if string inputs should result in an error and that was the fix (which is what the above sounds like to me), or if string inputs used to result in an error and that was the fix.